framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceScenes.java

```diff
@@ -283,6 +284,12 @@ public void updateFromFieldAssignment(
                             + lhs.getClass());
         }
 
+        // Do not attempt to infer types for temporary variables.
```

Comment:
It's unclear how this comment applies to the check being performed. Maybe something like "Do not attempt to infer types for variables with invalid names, which can be generated by temporary variables. See ...".